examples/webgpu_mrt.html

```diff
@@ -86,16 +86,16 @@
 					output: output,
 					normal: transformedNormalWorld.directionToColor(),
 					diffuse: diffuseColor,
-					emissive: emissive
+					velocity: velocity()
```

Comment:
@sunag Is there a way to refactor `VelocityNode` such that `velocity` can be used (meaning without `()` like with `output` or `emissive`)?

Reply:
I am moving beforeUpdate() to update() because projection matrix is updated after this, and adding first frame flag